webapp/locales/de.json

```diff
@@ -21,6 +21,7 @@
     }
   },
   "site": {
+    "thanks": "Danke!",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-30T11:30:26Z" title="Friday, August 30th 2019, 1:30:26 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -21,6 +21,7 @@
     }
   },
   "site": {
+    "thx": "Danke!",
```


No abbreviations please and be a little more verbose on the confirmation message.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T11:56:04Z" title="Friday, August 30th 2019, 1:56:04 pm +02:00">Aug 30, 2019</time>_

---


Yeah, I agree …